pkg/standalone/standalone.go

```diff
@@ -225,8 +226,7 @@ func makePodSourceConfig(kc *KubeletConfig) *config.PodConfig {
 		glog.Infof("Adding manifest url: %v", kc.ManifestURL)
 		config.NewSourceURL(kc.ManifestURL, kc.HttpCheckFrequency, cfg.Channel(kubelet.HTTPSource))
 	}
-
-	if kc.EtcdClient != nil {
+	if !reflect.ValueOf(kc.EtcdClient).IsNil() {
```

Comment:
kc.EtcdClient will be nil when kubelet is used standalone which is the case of its usage in the container-vm image. Or do I misunderstand your comment?